### ISSUE TITLE
Compass dosn't work on Crius aiop V1.1

### ISSUE THREAD

**Issue by Jogypilote** [MegaPirateNG/ardupilot-mpng#104]
Hi all, 
installed the latest version of MPNG 3.01 R4 on my Crius board. I followed the instructions from Arducopter and calibrated the compass several times, but the result is always the same: the compass pionts to ~330° doesn't matter which direction the copter faces. When I turn the copter the compass swings forth and back and stays at 330°.
First I thought the compass is defect but when I use the Multiwii software at the board the compass works perfectly.
Is there any idea what could be the problem?

Joerg

**Comment by wengwah** [comment]
Suggest that you terminal in and exe erase command follow by reset to default

**Comment by Jogypilote** [comment]
Thank you for your help.
I followed two times your instructions but the result is always nearly the same. Only the direction the compass points to after calibration changes. :-(

**Comment by thomasbutler** [comment]
I'm running the Crius v1.1 with MPNG 3.01 R4, and the compass works fine.  The copter has to be rotated on the x,y,and z axis in calibration, and manually add (that's what I did) the declination.  However, I have been updating my firmware periodically.  I've only been upgrading to versions I think will be of benefit (feature I really want).  I believe I went from 2.8R3 to 3.0.1R2 in October and I had to do a complete reset/erase of the flash in terminal mode.
